Makefile

```diff
@@ -0,0 +1,19 @@
+all: push
+
+# See pod.yaml for the version currently running-- bump this ahead before rebuilding!
+TAG = 0.1
+
+# TODO(random-liu): Change the project to google_containers.
+PROJ = google.com/noogler-kubernetes
```

Comment:
Maybe let @david-mcmahon know we are maintaining a new image there, but you should be set to push this to gcr.io/google-containers